packages/nimble-components/src/tabs/specs/scrollable-tabs.md

```diff
@@ -0,0 +1,106 @@
+# Scrollable tabs
+
+## Problem Statement
+
+There are some use-cases that call for many tabs, and in cases where the width of all of the tabs exceed the screen space, they will bleed off screen (and potentially make the page scrollable to see the other tabs). To prevent this, we will introduce "scrollable" tabs, which will enforce the tabs to be constrained to the given space of the `Tabs` component and offering buttons to allow a user to navigate through the whole set of tabs.
+
+### Out of scope of this HLD
+
+Any decision/API around limiting the width of a particular tab. This is ultimately an orthogonal concern to the UX this HLD is presenting.
+
+## Links To Relevant Work Items and Reference Material
+
+-   [Issue 1509: Scrollable Tabs](https://github.com/ni/nimble/issues/1509)
+-   [Figma design](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=11133-57220&node-type=instance&t=DlnrdRo7KWYlheFz-0)
+
+## Implementation / Design
+
+![Scrollable Tabs](./spec-images/scrollable-tabs.png)
+
+The following decisions were agreed to in the comments section in the linked issue above:
+
+-   No scrollbar will be shown
+-   Keyboard navigation will work as it does today, but when a tab that isn't fully in view gets focus it should be scrolled into view
+-   When a scroll button is pressed we will scroll either a full "page width" or the remainder of the available space, and tab focus is not changed.
+
+To accomplish this we will fork FAST's template in order to add the scroll buttons. A `ResizeObserver` will be used to track when the scrollable area exceeds the viewable area in order to show/hide the scroll buttons.
+
+There will be no new API to enable this behavior, and will simply be on by default.
+
+#### TabsToolbar
+
+Only the tabs themselves will be scrollable. The scroll buttons will only surround the tabs. If there is content being displayed in a `TabsToolbar` that has been slotted, and the tabs content is wide enough to merit showing the scroll buttons, the right scroll button will sit to the left of the `TabsToolbar` (example below).
+
+![Scrollable Tabs with Toolbar](./spec-images/scrollable-tabs-toolbar.png)
+
+### Keyboard interaction
+
+#### `PageUp/PageDown`
+
+Now that the tabs will be able to be scrolled, it makes sense to add a keyboard interaction that will perform this scroll without the need of using the buttons. When a tab in the tab list has focus, pressing `PageUp/PageDown` will perform the same scroll behavior as pressing the scroll buttons _and additionally_ jump the tab focus by the same page amount (or to the first/last tab). This is needed to make the scrolling of the tabs useful for a keyboard-only user.
```

Comment:
When you say "jump the tab focus", it would be good to clarify that for anchor tabs, the tab focus moves, but it shouldn't cause the focused tab to be activated. (Same is true for `Home/End`.) I believe that is the only difference between regular tabs and anchor tabs that applies to this HLD.